pytorch_lightning/loops/dataloader/prediction_loop.py

```diff
@@ -53,10 +53,7 @@ def num_dataloaders(self) -> int:
     @property
     def max_batches(self) -> List[int]:
         """The max number of batches this loop will run for each dataloader."""
-        max_batches = self.trainer.num_predict_batches
-        if isinstance(max_batches, int):
-            max_batches = [max_batches] * len(self.dataloaders)
```

Comment:
curious, are you removing this due to a redundancy, or because it is part of the fix? 
I suppose there is already logic in data_loading.py that takes care of this, so `isinstance(max_batches, int)` is never True?

Reply:
not a part of the fix, but it's redundant. was trying to fix some mypy issues and found this. num_pred_batches is always a list of ints.